docs/deployment/getting_started_locally.md

```diff
@@ -331,6 +331,42 @@ make kind2-down
 make kind-down
 ```
 
+## Alternative way to set up Garden and Seed leveraging `gardener-operator`
+
+Instead of starting Garden and Seed via `make kind-up gardener-up` you can also use `gardener-operator` to create your local dev landscape.  
```

Comment:
```suggestion
Instead of starting Garden and Seed via `make kind-up gardener-up`, you can also use `gardener-operator` to create your local dev landscape.  
```